l10n_mx_toponyms/hooks.py

```diff
@@ -0,0 +1,405 @@
+# -*- coding: utf-8 -*-
+
+from .models.helper_functions import get_date_int_and_date_udpate
+from .models.helper_functions import insert_into_ir_model_data, select_xmlid
+from .models.helper_functions import get_create_and_write_date
+from openerp import SUPERUSER_ID
+
+import csv
+import logging
+import os
+
+
+# Get a logger to work with
+_logger = logging.getLogger(__name__)
+
+
+try:
+    from unidecode import unidecode
+except ImportError:
+    _logger.debug('Cannot `import unidecode`.')
+
+try:
+    from openupgradelib.openupgrade import is_module_installed, load_data
+except ImportError:
+    _logger.debug('Cannot `import openupgradelib`.')
+
+
+def _load_county_data(cr):
+    """
+    Load examples into res_country_state_county, we are not using a normal xml
+    to accomplish this task due to it is really slow,
+    I know surely this is not the best way to do it but who cares :)
+    """
+    # Read csv from a speficic path
+    county_csv = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)),
+        'examples', 'l10n_mx_county.csv',
+    )
+    with open(county_csv, 'rt') as csvfile:
+        examples = csv.reader(csvfile, delimiter=',')
+        _ = examples.next()  # noqa: F841
```

Comment:
OK, try to see. You can also use the low level method https://github.com/odoo/odoo/blob/c60e43a9a407fd150a327281db841f6e9c81f266/odoo/models.py#L823